webapp/src/shared/utils/HandleRequestError.ts

```diff
@@ -0,0 +1,22 @@
+import { ResponseError } from '../generated';
+
+export const handleRequestError = (
+  error: unknown,
+  msg: string,
+  loggerParam?: (arg0: any) => void,
+) => {
+  const logger = loggerParam ? loggerParam : (arg0: any) => console.error(arg0);
```

Comment:
Pitiminí:

```suggestion
  const logger = loggerParam ?? (arg0: any) => console.error(arg0);
```

Reply:
It doesn't work if you don't put parenthesis around the function on the right. ??